python/modules/csv_managment.py

```diff
@@ -74,6 +74,7 @@ def read_csv(path, mark, dict_local = {}, key_name = ""):
 def write_csv(path, deltimer, data = None, key = ""):
```

Comment:
I have to change this across all files, but wanted to do whole module at once not just one function.